dokka-subprojects/plugin-base-frontend/postcss.config.js

```diff
@@ -5,16 +5,6 @@
 module.exports = () => ({
   plugins: [
     require('postcss-import'),
-    require('postcss-preset-env')({
```

Comment:
this seems to be not working and only producing a waring about unknown postcss features options so I just removed it